python/lsst/ctrl/oods/butlerAttendant.py

```diff
@@ -85,13 +78,15 @@ def __init__(self, config, csc=None):
             on_ingest_failure=self.on_ingest_failure,
             on_metadata_failure=self.on_metadata_failure,
         )
+        define_visits_config = DefineVisitsTask.ConfigClass()
+        define_visits_config.groupExposures = "one-to-one"
```

Comment:
Note for the future: this may need to be configurable (or at least changed) to `one-to-one-and-by-counter`.